oxidation/client/src/services/storageManager.ts

```diff
@@ -58,7 +59,9 @@ export class StorageManager {
     Object.keys(data).forEach((slug, _) => {
       if (data[slug] && data[slug].lastLogin) {
         deviceData[slug] = {
-          lastLogin: new Date(data[slug].lastLogin)
+          // Without the `setZone`, luxon ignores the zone in the string because
+          // it wouldn't be fun or well made otherwise
```

Comment:
update comment with doc or issue link, delete the 'fun or well made otherwise' part